core/tests/ts-integration/tests/fees.test.ts

```diff
@@ -54,12 +56,41 @@ testFees('Test fees', () => {
     let tokenDetails: Token;
     let aliceErc20: zksync.Contract;
 
-    beforeAll(() => {
+    beforeAll(async () => {
         testMaster = TestMaster.getInstance(__filename);
         alice = testMaster.mainAccount();
 
         tokenDetails = testMaster.environment().erc20Token;
         aliceErc20 = new ethers.Contract(tokenDetails.l1Address, zksync.utils.IERC20, alice.ethWallet());
+
+        const mainWallet = new zksync.Wallet(
+            testMaster.environment().mainWalletPK,
+            alice._providerL2(),
+            alice._providerL1()
+        );
+
+        const bridgehub = await mainWallet.getBridgehubContract();
+        const chainId = testMaster.environment().l2ChainId;
+        const baseTokenAddress = await bridgehub.baseToken(chainId);
```

Comment:
Does it need to be behind `if (!isEthChain)`? As we want deposit to be made only if the chain is on custom base token